rpc/session.go

```diff
@@ -0,0 +1,21 @@
+package rpc
+
+import (
+	"net"
+	"net/rpc"
+	"net/rpc/jsonrpc"
+)
+
+type Session struct {
+	*rpc.Client
+}
+
+func NewSession(conn net.Conn) *Session {
+	return &Session{
+		Client: rpc.NewClientWithCodec(jsonrpc.NewClientCodec(conn)),
+	}
+}
+
+func (s *Session) Close() {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method Session.Close should have comment or be unexported